--- index2.ts	2024-05-14 12:04:46.778079121 +0300
+++ index.ts	2024-05-14 12:05:10.486886140 +0300
@@ -138,13 +138,6 @@
 		// Wait for system time to be synced for up to 10 seconds before proceeding
 		await this.waitForSystemTime(10)
 
-		// We need to forcefully clean Docker state before being able to safely continue
-		// If an existing container is listening on port 80 we'll crash, if an old version
-		// of Umbrel wasn't shutdown properly, bringing containers up can fail.
-		await this.apps
-			.cleanDockerState()
-			.catch((error) => this.logger.error(`Failed to clean Docker state: ${(error as Error).message}`))
-
 		// Initialise modules
 		await Promise.all([this.apps.start(), this.appStore.start(), this.server.start()])
 	}
